deploy/test-environments/delete_env.sh

```diff
@@ -24,14 +24,14 @@ function delete_environment() {
     tfstate="./$ENV-terraform.tfstate"
 
     # Copy state file
-    if aws s3 cp $BUCKET/"$ENV"/terraform.tfstate "$tfstate"; then
+    if aws s3 cp "$BUCKET"/"$ENV"/terraform.tfstate "$tfstate"; then
```

Comment:
```suggestion
    if aws s3 cp "$BUCKET/$ENV/terraform.tfstate" "$tfstate"; then
```

Slightly more readable IMO